beaker_kernel/lib/llm_tasks.py

```diff
@@ -0,0 +1,57 @@
+from dataclasses import dataclass
```

Comment:
I think it would be more consistent if this file was called `agent_tasks.py` to better fit with the more of the code. The "Agent" is always the one doing things with the LLM instead of the LLM doing things, if that makes sense.